src/renderer/components/AgentBase.vue

```diff
@@ -331,7 +327,7 @@ export default {
     AgentStaticConnections,
     AgentMyCredentialsList,
     AgentTrust,
-    Presentations,
+    PresentationsTab,
```

Comment:
Same comment as on #83, I think following the naming convention of just "Presentations" is preferable, especially considering that we intend to move away from the tab setup ASAP.